packages/react-reconciler/src/ReactFiberThrow.new.js

```diff
@@ -255,21 +255,21 @@ function throwException(
           wakeables.add(wakeable);
         }
 
-        // If the boundary is outside of blocking mode, we should *not*
+        // If the boundary is in legacy mode, we should *not*
         // suspend the commit. Pretend as if the suspended component rendered
         // null and keep rendering. In the commit phase, we'll schedule a
         // subsequent synchronous update to re-render the Suspense.
         //
         // Note: It doesn't matter whether the component that suspended was
-        // inside a blocking mode tree. If the Suspense is outside of it, we
+        // inside a legacy tree. If the Suspense is outside of it, we
```

Comment:
blocking mode tree !== legacy tree

Reply:
Meaning, I think this comment should be the other way around?

Reply:
Sometimes blocking mode matches legacy semantics and sometimes it matches concurrent mode. I thought that this was a case where it was grouping blocking and legacy together, but maybe I'm reading the comment wrong?

Reply:
Reading this again, I think you're right. Updated.